src/stylesheets/_accessibility.scss

```diff
@@ -0,0 +1,10 @@
+@mixin visually-hidden {
+    border: 0;
```

Comment:
Isn't there a less hacky way to do this?

Reply:
This has been the de facto way to visually hide content for quite some time.